azure-service-bus-management/1.0/modules/ROOT/pages/azure-service-bus-management-config-topics.adoc

```diff
@@ -0,0 +1,10 @@
+= Azure Service Bus Management 1.0 Additional Configuration Topics - Mule 4
+
+Anypoint Azure Service Management connector cannot perform a connectivity test at startup using the `Test Connection` configuration option when you restrict access to your resources
```

Comment:
Anypoint Connector for Azure Service Bus Management (Azure Service Bus Management Connector)